helm-charts/charts/warehouse/values.yaml

```diff
@@ -0,0 +1,207 @@
+# Ensure that the warehouse instance name differs from the starrocks instance name to avoid conflicts. For example, both
+# will generate configmaps using the 'nameOverride' as a prefix to the configmap name, followed by a specific suffix.
+nameOverride: "kube-starrocks"
+
+# TimeZone is used to set the environment variable TZ for pod, with Asia/Shanghai as the default.
+timeZone: Asia/Shanghai
+
+# This configuration is used to modify the root password during initial deployment.
+# After deployment is completed, it won't take effect to modify the password here and to do a `helm upgrade`.
+# It also supports providing secret name that contains password, using the password in the secret instead of the plaintext in the values.yaml.
+# When both password and passwordSecret are set, only passwordSecret takes effect.
+initPassword:
+  enabled: false
+  # The name of the secret which contains password, the key in the secret is "password", and you should create it first.
+  passwordSecret: ""
+
+# This configuration is used to integrate with external system DataDog.
+# You can enable the integration by setting the enabled to true, e.g. datalog.log.enabled=true will enable datadog agent
+# to collect the log.
+datadog:
+  log:
+    enabled: false
+    # besides the attributes you added, chart will append "source" and "service" attributes to the log config.
+    # see https://docs.datadoghq.com/containers/kubernetes/log/?tab=operator for more details.
+    logConfig: '{}' # e.g. '{"app": "starrocks", "tags": ["aa", "bb"]}'
+  metrics:
+    enabled: false
+
+# The name of starrocks cluster. Note: the starrocks cluster should be created before the warehouse by operator.
+starRocksCluster:
+
+# spec for compute node, compute node provide compute function.
+starrocksCnSpec:
```

Comment:
is it possible to rename these `cn` related configuration items?

Reply:
Ok, I forgot to change the helm chart part.